plugins/woocommerce/src/Admin/PluginsInstallLoggers/AsynPluginsInstallLogger.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\PluginsInstallLoggers;
+
+/**
+ * A logger to log plugin installation progress in real time to an option.
+ */
+class AsynPluginsInstallLogger implements PluginsInstallLogger {
```

Comment:
Nit: Perhaps, we should use `Async`?

```suggestion
class AsyncPluginsInstallLogger implements PluginsInstallLogger {
```